.murdock

```diff
@@ -236,6 +244,34 @@ compile() {
         make -C${appdir} clean all test-input-hash -j${JOBS:-4}
     RES=$?
 
+    test_hash=$(test_hash_calc "$BINDIR")
```

Comment:
depending on whether we want to go with this change: further down is another invocation to `test_hash_calc`. That can be replaced with `${test_hash}`